README.md

```diff
@@ -1,3 +1,14 @@
+Some assumptions/decisions made:
+    1. For modularizing the AccountTrigger code, the assignment isn't to extend the TriggerHandler, but to simply pull most of the logic out of the AccountTrigger.
+    2. Since that leaves the AccountTrigger simply calling methods in the Helper, I'm combining the 3 identical If Statements into 1, reducing code and operation.  Not sure if that's good practice or not, just making a call on this.
```

Comment:
What you did is also what I would do. I think they are better grouped together by operation.